components/calendar/src/any_calendar.rs

```diff
@@ -1628,43 +1628,9 @@ mod tests {
     }
 
     #[test]
-    fn test_any_construction() {
+    fn buddhist() {
```

Comment:
praise: good call, this is neater, and we're not really making use of having all the calendars in one place